src/libpmemfile-posix/inode.c

```diff
@@ -80,6 +80,8 @@ vinode_ref(PMEMfilepool *pfp, struct pmemfile_vinode *vinode)
 	return vinode;
 }
 
+#define INITIAL_NBUCKETS 2
```

Comment:
It was @pbalcer, who asked me to do it. I don't know how to link to this comment. Look at GitHub interface.